build-system/pr-check.js

```diff
@@ -361,22 +361,22 @@ function main(argv) {
   }
 
   if (process.env.BUILD_SHARD == "integration_tests") {
-    if (buildTargets.has('RUNTIME')) {
-      command.buildRuntime();
-      command.runIntegrationTests();
```

Comment:
I think this will also disable integration tests on Saucelabs. Check out this sample master build that runs `pr-check.js`: https://travis-ci.org/ampproject/amphtml/jobs/235767902